<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chargeritem</class>
 <widget class="QFrame" name="chargeritem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <widget class="QFrame" name="chargerItemWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>550</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="connectorId">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>35</y>
      <width>280</width>
      <height>82</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectorType">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>97</y>
      <width>280</width>
      <height>65</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>CCS</string>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="connectorPower">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>280</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>120 kW</string>
    </property>
   </widget>
   <widget class="QFrame" name="plugIcon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>213</y>
      <width>280</width>
      <height>116</height>
     </rect>
    </property>
   </widget>
   <widget class="ProgressBarWidget" name="progressBar" native="true">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>385</y>
      <width>151</width>
      <height>15</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>414</y>
      <width>280</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>5%</string>
    </property>
   </widget>
   <widget class="QWidget" name="chargerItemHoverWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>280</width>
      <height>550</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#hoverWidget {
    background-image: url(:/assets/images/ccs_interface_background_selected.png);
    background-repeat : no-repeat;
    background-position : center;
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="energyAndPowerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>447</y>
      <width>189</width>
      <height>55</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>188</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="powerLabel">
        <property name="minimumSize">
         <size>
          <width>57</width>
          <height>46</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>46</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true">Power
68 kW</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="socketIcon">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>45</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="energyLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">Energy
10 kWh</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>chargerItemHoverWidget</zorder>
   <zorder>connectorPower</zorder>
   <zorder>plugIcon</zorder>
   <zorder>connectorType</zorder>
   <zorder>connectorId</zorder>
   <zorder>progressBar</zorder>
   <zorder>progressLabel</zorder>
   <zorder>energyAndPowerWidget</zorder>
  </widget>
  <widget class="QWidget" name="reservationOrFaultedWidget" native="true">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>35</y>
     <width>230</width>
     <height>510</height>
    </rect>
   </property>
   <widget class="QWidget" name="reservationOrFaultedGrayBackgroundWidget" native="true">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>1</y>
      <width>210</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="chargerItemNotificationIconAndLabelWidget" native="true">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>295</y>
      <width>230</width>
      <height>200</height>
     </rect>
    </property>
    <widget class="QWidget" name="chargerItemNotificationIconWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>14</y>
       <width>230</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="reservationOrFaultedLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>230</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProgressBarWidget</class>
   <extends>QWidget</extends>
   <header>progressbarwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
